react/components/AutocompleteResults/styles.css

```diff
@@ -0,0 +1,27 @@
+.resultsList {
+  list-style-type: none;
+  max-height: 60vh;
+  overflow-y: auto;
+  top: 100%;
+}
+
+.resultsList a,
+.resultsList div {
+  text-decoration: none;
+  color: #727273;
+}
+
+.resultsList a:hover {
+  background-color: #f2f4f5;
+}
+
+.resultsItemImage {
+  width: 50px;
+}
+
+@media only screen and (max-width: 49rem) {
+  .resultsList {
+    margin: 0px;
```

Comment:
There is some classes in tachyons for this too with breakpoints, I don't know if exact breakpoint is there, but you could check

Reply:
Unfortunately, this breakpoint is very specif. I was not able to find an equivalent tachyons class